versioned_docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md

```diff
@@ -496,7 +496,7 @@ GRANT ALL PRIVILEGES ON centreon.* TO '@MARIADB_CENTREON_USER@'@'@DATABASE_MASTE
 GRANT ALL PRIVILEGES ON centreon_storage.* TO '@MARIADB_CENTREON_USER@'@'@DATABASE_MASTER_IPADDR@';
 ```
 
-Optionnaly, you can allow these privileges to be used from Central Cluster. It will make some administration scripts runnable from every nodes.
+Optionally, you can allow these privileges to be used from Central Cluster.  This will make some administration scripts runnable from every node.
```

Comment:
```suggestion
Optionally, you can allow these privileges to be used from the Central Cluster.  This will make some administration scripts runnable from every node.
```